src/main/java/org/opentripplanner/middleware/controllers/api/OtpRequestProcessor.java

```diff
@@ -122,9 +123,30 @@ private static void handlePlanTripResponse(Request request, OtpDispatcherRespons
         long tripStorageStartTime = DateTimeUtils.currentTimeMillis();
 
         Auth0Connection.checkUser(request);
-        Auth0UserProfile profile = Auth0Connection.getUserFromRequest(request);
+        RequestingUser requestingUser = Auth0Connection.getUserFromRequest(request);
+        if (requestingUser == null) {
+            return;
+        }
```

Comment:
Can you add a comment here about what's happening?

Reply:
Updated.

Reply:
I think the comment for checking if requestingUser == null might have been skipped, but I think it would still be useful.

Reply:
Updated.